api/src/pcapi/scripts/format_user_phone_numbers.py

```diff
@@ -0,0 +1,147 @@
+import argparse
+from datetime import datetime
+import logging
+import os
+import warnings
+
+from sqlalchemy.orm import load_only
+
+
+warnings.filterwarnings("ignore")
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_users(from_, to):  # type: ignore [no-untyped-def]
+    users = (User.query.options(load_only("id", "phoneNumber")).order_by(User.id))[from_:to]
+    return users
```

Comment:
Si je comprends bien, l'idée est de charger tous les utilisateurs, les trier par id et de ne prendre que ceux de l'indice `from_` à l'indice `to`.

Tu as testé en staging pour être sûr que ça passe ?

Suggestion :

1. supprimer `[from_:to]`
2. Ajouter un `yield_per`

Et si tu as besoin de regrouper les utilisateurs par paquets de N, tu as la fonction [get_chunks](https://github.com/pass-culture/pass-culture-main/blob/master/api/src/pcapi/utils/chunks.py#L9) qui peut t'aider.

Reply:
alors le slicing est fait sur une query SQLAlchemy alors ça va juste ajouter un `LIMIT ... OFFSET` sur la requête et ne charger qu'un "petit" nombre d'utilisateurs à chaque fois.

j'ai bien testé en staging et ça passe.